### ISSUE TITLE
Policy prefix-set is not take effect in vpnv4 prefix

### ISSUE THREAD

**Issue by grandhuhu** [osrg/gobgp#2115]

**Comment by grandhuhu** [comment]
I saw the code in policy.go line 1454 and function is PrefixCondition. In 1466 line:
 `	case bgp.RF_IPv4_UC:
		masklen = path.GetNlri().(*bgp.IPAddrPrefix).Length
		key = keyf(path.GetNlri().(*bgp.IPAddrPrefix).Prefix, int(masklen))
	case bgp.RF_IPv6_UC:
		masklen = path.GetNlri().(*bgp.IPv6AddrPrefix).Length
		key = keyf(path.GetNlri().(*bgp.IPv6AddrPrefix).Prefix, int(masklen))`
Is it not support for vnpv4 prefix?

**Comment by fujita** [comment]
You are right. What policy representations are used for vpnv4 prefix in other bgp implementations?
